skgstat/Variogram.py

```diff
@@ -42,18 +43,19 @@ def __init__(self,
                  ):
         r"""Variogram Class
 
-        Note: The directional variogram estimation is not re-implemented yet.
-        Therefore the parameters is-directional, azimuth and tolerance will
-        be ignored at the moment and can be subject to changes.
-
         Parameters
         ----------
-        coordinates : numpy.ndarray
+        coordinates : numpy.ndarray, MetricSpace
+            .. versionchanged:: 0.5.0
+                now accepts MetricSpace
             Array of shape (m, n). Will be used as m observation points of
             n-dimensions. This variogram can be calculated on 1 - n
             dimensional coordinates. In case a 1-dimensional array is passed,
             a second array of same length containing only zeros will be
             stacked to the passed one.
+            For very large datasets, you can use the MetricSpace class, that
+            will only calculate distances within the maximum lag in a 
+            sparse matrix.
```

Comment:
It is semi-correct.

I'd say something like:


For very large datasets, you can set maxlag to only calculate distances within the maximum lag in a sparse matrix. Alternatively you can supply a MetricSpace (optionally with a max_dist set for the same effect). This is useful if you're creating many different variograms for different measured parameters that are all measured at the same set of coordinates, as distances will only be calculated once, instead of once per variogram.

Reply:
A similar comment might be useful in the parameter doc for `Kriging`.